src/traces/carpet/defaults.js

```diff
@@ -21,6 +21,8 @@ module.exports = function supplyDefaults(traceIn, traceOut, dfltColor, fullLayou
         return Lib.coerce(traceIn, traceOut, attributes, attr, dflt);
     }
 
+    traceOut._clipPathId = 'clip' + traceOut.uid + 'carpet';
```

Comment:
Oh wow… Good catch! Yeah, I can imagine that's a subtle half-border-width-effect. 👍